lib/webtask.js

```diff
@@ -383,14 +387,6 @@ exports.ensure_authorized = function (admin, ctx) {
     };
 };
 
-var lru; // LRU cache of token revocation information
-
-exports.clear_revocation_lru = function (jti) {
```

Comment:
This was only used by the proxy to remove the jti from the lru cache following revocation using webtask call. Now the dynamo call and removal from LRU cache is all encapsulated in one method (`revoke` above).